packages/infra/lib/api-stack.ts

```diff
@@ -542,6 +554,8 @@ export class APIStack extends Stack {
     outboundDocumentQueryLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
     outboundDocumentRetrievalLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
     fhirToBundleLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
+    patientCreateLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
+    patientQueryLambda.addEnvironment("API_URL", `http://${apiDirectUrl}`);
```

Comment:
should be name this discovery lambda to reserver query lambda for doc query (when/if we add it).